Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Data/Security/1040-TPDM-Ensure-people-claim-has-no-parent-resource-claim.sql

```diff
@@ -0,0 +1,13 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+-- Note: This step undoes the parent resource claim assignment that resulted in people being placed under EducationOrganizations.
+-- This was previously (inadvertently) performed in 1010-TPDM-ResourceClaims.sql, starting at line 2044.
+
+PRINT 'Ensuring that the ''people'' resource claim does not have a parent in the hierarchy.'
+
+UPDATE  dbo.ResourceClaims
```

Comment:
As we follow in other scripts it will be good  if we have if exists before updating the ParentResourceClaimId value for dbo.ResourceClaims table .  Same thing for pgsql also

Reply:
Well, adding that check really doesn't realistically change anything... I can't imagine a deployment where this resource claim doesn't exist, and if it doesn't, there would just be nothing matched in the update.

However, I will add guard conditions to both SQL Server and PostgreSQL versions with an additional check for the a ParentResourceClaimId that is _not_ null... to avoid executing the query at all if the data is already in the desired final state.

Reply:
Agreed